lib/diagnostic.rb

```diff
@@ -105,15 +121,23 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < cat
+  EATS = Carnivorous.new
+
```

Comment:
remember that we can include mixins by using the `include` keyword
so `include Carnivorous`